staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -370,7 +370,7 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		// Generic API servers have no inherent long-running subresources
 		LongRunningFunc:           genericfilters.BasicLongRunningRequestCheck(sets.NewString("watch"), sets.NewString()),
 		lifecycleSignals:          lifecycleSignals,
-		StorageObjectCountTracker: flowcontrolrequest.NewStorageObjectCountTracker(lifecycleSignals.ShutdownInitiated.Signaled()),
```

Comment:
is there any change in the shutdown process stage where this runs? I mean, can this be assuming it was running in a certain order that we may be changing here and cause some undesirable side effect?